Docs/Configuration.tex

```diff
@@ -1408,12 +1404,12 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   \end{itemize}
 
   This quirk requires \texttt{ProvideCustomSlide} to also be enabled
-  and generally needs \texttt{AvoidRuntimeDefrag} to work correctly.
-  Hibernation is not supported when booting with a relocation block
-  (but relocation block is not always used when the quirk is enabled).
+  and typically needs \texttt{AvoidRuntimeDefrag} to be set for this to
```

Comment:
Any reason to change `generally` to `typically`? They mean similar things but I do wonder what is the meaning change behind.

Reply:
Those are two difficult words to explain the very subtle difference.

"Generally this or that" comes across a bit passive and loose when you are supposed on being definitive and precise.

- Typically suggests it is the character/nature of Quirk to need AvoidRuntimeDefrag. 
  - "While Quirk may sometimes not need AvoidRuntimeDefrag, it almost certainly always does."
- Generally is just ever so slightly less strong an indication to the reader. 
  - "Quirk usually needs AvoidRuntimeDefrag but I really don't have a clue when it will."

Not a perfect explanation as difference is subtle as said.